src/main/java/frc/robot/Constants.java

```diff
@@ -104,6 +103,25 @@ public static final class ModuleConstants {
         public static final int ANGLE_CURRENT_LIMIT = 25;
     }
 
+    public static final class IntakeConstants {
+        public static final Gains PID_GAINS = new Gains(0, 0, 0);
+        public static final Gains FF_ANGLE_GAINS = new Gains(0, 0, 0);
+
+        public enum INTAKE_ANGLE {
+            GROUND(0),
+            AMP(0),
+            HUMAN_PLAYER(0),
+            TO_SHOOTER(0),
+            IDLE(0);
```

Comment:
the `shooter` and `idle` should be the same one, actually just delete the idle, because the intake will rest in the shooter position when not in use (so its ready to transport to shooter, and so the motor doesn't have to hold it, because it rest on the mechanical stop)..

also change `TO_SHOOTER` to just `SHOOTER`